stories/documentation/listings/index-table/index-table-empty-state.stories.ts

```diff
@@ -0,0 +1,53 @@
+import { Meta, StoryFn } from '@storybook/angular';
+
+interface IndexTableEmptyStateStory {
+
+}
+
+export default {
+	title: 'Documentation/Listings/Index Table/Empty State',
+	argTypes: {
+		
+	},
+} as Meta;
+
+function getTemplate(args: IndexTableEmptyStateStory): string {
+	return `
+	<table class="indexTable">
```

Comment:
For empty states, the table is here for decorative purposes only, I think it should be given a `role="presentation"`.